docs/CHILI-GraFx/browser-support/index.md

```diff
@@ -0,0 +1,49 @@
+# Browser support for CHILI GraFx
+
+For all users: understand which web browsers are supported when using CHILI GraFx apps and services.
+
+## Overview
+
+CHILI GraFx is a cloud-based platform with a browser-based user interface. While rendering, data processing, and automation happen in the cloud, the user-facing apps — like GraFx Studio and the GraFx Experience — run in the browser and rely on modern web technologies.
+
+We distinguish between two levels of browser support:
+
+- **First-party support** means the browser is actively tested and fully supported. If you encounter issues, we treat them as bugs and commit to fixing them.
+- **Best-effort support** means the browser is not part of our test suite. Most features will likely work, but there’s no guarantee. If issues occur, we evaluate them case by case.
+
+Browsers outside these categories are not officially supported and may not function correctly.
+
+## Supported browsers
+
+### ✅ First-party support
+
+We guarantee full compatibility and run automated tests for "Chromium based" browsers:
```

Comment:
I'd not talk about automated tests it's too specific (and not even true as we only run them on chromium itself) also i'd be careful with explicitly mentioning other chromium based browsers, this implies that we support every browser running chromium also very obscure once.